Tests/SentryTests/Integrations/Performance/Network/SentryNetworkTrackerTests.swift

```diff
@@ -448,56 +448,6 @@ class SentryNetworkTrackerTests: XCTestCase {
         assertOneSpanCreated(transaction)
     }
     
-    func test_AddTraceHeader() {
-        let sut = fixture.getSut()
-        SentrySDK.currentHub().scope.span = SentryTracer(transactionContext: TransactionContext(name: "SomeTransaction", operation: "SomeOperation"), hub: nil)
-        let headers = sut.addTraceHeader([:])
-        XCTAssertEqual(headers?.count, 1)
-        XCTAssertNotNil(headers?["baggage"])
-
-        let decodedBaggage = SentrySerialization.decodeBaggage(headers?["baggage"] ?? "")
-        XCTAssertEqual(decodedBaggage.count, 5)
-    }
-
-    func test_AddTraceHeader_AppendOriginalBaggage() {
-        let sut = fixture.getSut()
-        SentrySDK.currentHub().scope.span = SentryTracer(transactionContext: TransactionContext(name: "SomeTransaction", operation: "SomeOperation"), hub: nil)
-        let headers = sut.addTraceHeader(["baggage": "key1=value"])
-        XCTAssertEqual(headers?.count, 1)
-        XCTAssertNotNil(headers?["baggage"])
-
-        let decodedBaggage = SentrySerialization.decodeBaggage(headers?["baggage"] ?? "")
-        XCTAssertEqual(decodedBaggage.count, 6)
-    }
-
-    func test_RemoveExistingBaggageHeader_WhenNoSpan() {
-        let sut = fixture.getSut()
-        let headers = sut.addTraceHeader(["a": "a", "baggage": "key=value,sentry-trace_id=sentry-trace_id,sentry-release=abc"])
-        XCTAssertEqual(headers?.count, 2)
-        XCTAssertEqual(headers?["baggage"], "key=value")
-    }
-
-    func test_RemoveExistingBaggageHeader_WhenNoSpan_NoEmptyBaggage() {
-        let sut = fixture.getSut()
-        let headers = sut.addTraceHeader(["a": "a", "baggage": "sentry-trace_id=sentry-trace_id,sentry-release=abc"])
-        XCTAssertEqual(headers?.count, 1)
-        XCTAssertNil(headers?["baggage"])
-    }
-    
-    func test_AddTraceHeader_NoTransaction() {
-        let sut = fixture.getSut()
-        let headers = sut.addTraceHeader([:])
-        XCTAssertEqual(headers?.count, 0)
-    }
-    
-    func test_AddTraceHeader_TrackingDisabled() {
-        let sut = fixture.getSut()
-        sut.disable()
-        let headers = sut.addTraceHeader([:])
```

Comment:
`m`: I think those two test cases are great. Why not keep them with the new code?

Reply:
Done